vector/src/main/java/im/vector/app/features/analytics/AnalyticsConfig.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2021 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.analytics
+
+import im.vector.app.BuildConfig
+import im.vector.app.features.analytics.log.analyticsTag
+import timber.log.Timber
+
+data class AnalyticsConfig(
+        val postHogHost: String,
+        val postHogApiKey: String
+) {
+    companion object {
+        /**
+         * Read the analytics config from the Build config
+         */
+        fun getConfig(): AnalyticsConfig? {
```

Comment:
not for this PR, eventually we might want to wrap and inject the `BuildConfig` rather than directly accessing it (will enable the config to be shared across modules)

Reply:
I have updated the way developer can configure the analytics in c1438f0a65758d79581bd999e367113b8001cf1a